### PR TITLE
13.0 add hr timesheet portal

### PR DESCRIPTION
**Pull request by KKamaa**
This is an addition or rather a port from [PR](https://github.com/OCA/timesheet/pull/430) **_hr_timesheet_portal_**

### CODE REVIEW

**Comment by bosd** [comment]
@KKamaa Can you fix Pre-Commit?

**Comment by KKamaa** [comment]
@bosd will do

**Comment by KKamaa** [comment]
@bosd This error is not related to my changes check the pre-commit logs. I ran this on my local and all pass, see below attached screenshot.

image

**Comment by bosd** [comment]
Looks like there is an issue in this repo.
@alexey-pelykh Can you help?

**Comment by KKamaa** [comment]
@alexey-pelykh Hey can you maybe take a look at this

**Comment by alexey-pelykh** [comment]
@KKamaa try to rebase on 13.0 branch, likely that would help. Also, please ensure that commits are as suggested by guidelines. Also, since #430 has been merged, I would guess you should redo the migration

**Comment by KKamaa** [comment]
@thomaspaulb this could be closed can we merge it?

**Comment by nimarosa** [comment]
@KKamaa if you don't get all checks green (pre-commit) is failing, we can't merge. 

Please check pre-commit logs to see what is missing to make it pass.

**Comment by nimarosa** [comment]
I will rebase it for you to see updated checks 

/ocabot rebase

**Comment by bosd** [comment]
Looks like a @timesheet-maintainer needs to update this repo.
How can we find out who is the maintainer of this repo??

copied from another thread how to do it:
You can follow the guide described in README here:

https://github.com/OCA/oca-addons-repo-template

The thing is to let all default answers by doing `copier update`


Edit: Having the same issue with this repo in: https://github.com/OCA/timesheet/pull/474#issuecomment-1208003851

**Comment by bosd** [comment]
@nimarosa Do you have the super powers to update the repo template?

**Comment by nimarosa** [comment]
@bosd yes it's happening in many repos. I will update the templates today and then we rebase.

**Comment by bosd** [comment]
@nimarosa , @KKamaa 
Repo template is updated. This one can be rebased now

File: hr_timesheet_portal/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Hunki Enterprises BV
+# Copyright 2021 Sunflower IT
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Timesheet portal (editable)",
+    "summary": "Fill in timesheets via the portal",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Website",
+    "website": "https://github.com/OCA/hr-timesheet",
```
Comment (bosd) [inline hr_timesheet_portal/__manifest__.py:10]: Thanks for this addition. Please attend to:
```
-    "website": "https://github.com/OCA/hr-timesheet",
+    "website": "https://github.com/OCA/timesheet",`
```


Then tests should pass. ✨

**Review by bosd — CHANGES_REQUESTED** [review]

File: hr_timesheet_portal/__manifest__.py (hunk 1 above)
Comment (bosd) [reply hr_timesheet_portal/__manifest__.py:10]: @KKamaa Can you attend to this?

**Pull request closed without merging**